ledger/eval_test.go

```diff
@@ -452,6 +455,7 @@ func testEvalAppGroup(t *testing.T, schema basics.StateSchema) (*BlockEvaluator,
 	defer l.Close()
 
 	newBlock := bookkeeping.MakeBlock(genesisInitState.Block.BlockHeader)
+	newBlock.RewardsState = genesisInitState.Block.RewardsState
```

Comment:
@tsachiherman , as part of moving SpecialAddresses into the BlackEvaluator, I needed to add this to several tests - it copies the RewardState over from genesis to the next block.  That's probably not exactly how that should work, but it was sufficient for the test (the new block needed the addresses from the old).  Let me know if you think something else should be done instead.